backend/app/agents/metabase_agent.py

```
@@ -210,23 +217,29 @@ def generate_sql(prompt: str) -> str:
 # Tool: Create Metabase card (async, uses real API)
 @function_tool
 async def create_card(
-        sql: str,
-        name: str,
-        viz_type: str = "table"
+    sql: str,
+    name: str,
+    viz_type: str = "table",
+    db_name: str = None
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)